wrappers/services.go

```diff
@@ -284,7 +299,8 @@ func StartServices(apps []*snap.AppInfo, disabledSvcs []string, flags *StartServ
 		if servicesStarted {
```

Comment:
do I read correctly that the prexisting StartServices tests are not affected by this refactors?

Reply:
yes that's correct, all unit tests that cover this code-path either don't test with socket activations, or does not test with multiple socket-activations (and thus not actually see any change as the sorting of the socket units is the thing affected).

Reply:
yes, no tests were affected by it